### PR TITLE
Add a couple of angle helpers to deal with degrees and radians

### PR DESCRIPTION
**Pull request by elland**
Passing tests

### CODE REVIEW

**Comment by 3lvis** [comment]
We need to get this CI working soon.

**Comment by 3lvis** [comment]
Nice addition, @Elland :clap:

**Comment by elland** [comment]
🎉

File: Sources/Math.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Foundation
```
Comment (3lvis) [inline Sources/Math.swift:1]: WTF

Comment (elland) [reply Sources/Math.swift:1]: That's not needed right?

Comment (3lvis) [reply Sources/Math.swift:1]: M_PI is part of Darwin/Foundation.

Comment (3lvis) [reply Sources/Math.swift:1]: http://stackoverflow.com/questions/15231466/whats-the-difference-between-pi-and-m-pi-in-objc

screen shot 2016-10-27 at 23 04 03

Comment (3lvis) [reply Sources/Math.swift:1]: Would be nice to keep this library without any imports.

Comment (elland) [reply Sources/Math.swift:1]: Yeah, let's move this to SweetFoundation instead?

**Pull request merged**